Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wk0-starter"
 version = "0.1.0"
-authors = ["Alex Ozdemir <aozdemir@hmc.edu>"]
+authors = ["Adam Dulnap <adunlap@hmc.edu>"]
```

Comment:
Nice! You found the Cargo.toml!

Also, I want to meet this Adam Dulnap kid -- seems pretty cool!